tests/visualization/test_graphviz.py

```diff
@@ -150,3 +149,36 @@ def test_filename(self):
         self.assertTrue(os.path.isfile("test_graphviz_filename.svg"))
         if not SAVE_IMAGES:
             self.addCleanup(os.remove, "test_graphviz_filename.svg")
+
+    def test_escape_sequences(self):
```

Comment:
I wonder if the escaping makes all variables have quotes by default

Reply:
yes, also I feel removing quotes will not be ideal , I will update the expected result

Reply:
As long as it doesn't break the graphs I am fine with either

Reply:
It doesnt, I am trying to see if have quotes is better than not having them but since we are handling special characters it shouldn't be a problem